data/json/items/armor/belts.json

```diff
@@ -151,6 +153,7 @@
     "description": "Black leather belt used by police officers.  It has several pouches and a holder for a baton.",
     "weight": "250 g",
     "volume": "2 L",
+    "max_item_length": "70 cm",
```

Comment:
I think `max_item_length` should go inside of `pocket_data`.

Reply:
yep, jumping on it rn

Reply:
I fixed the problem